blobstore/src/test/java/org/jclouds/blobstore/integration/internal/BaseContainerIntegrationTest.java

```diff
@@ -553,7 +553,7 @@ protected void addTenObjectsUnderPrefix(String containerName, String prefix) thr
 
    protected void awaitConsistency() {
       if (view.getConsistencyModel() == ConsistencyModel.EVENTUAL) {
-         Uninterruptibles.sleepUninterruptibly(10, TimeUnit.SECONDS);
+         Uninterruptibles.sleepUninterruptibly(20, TimeUnit.SECONDS);
```

Comment:
tl;dr we should not increase the timeout since this increases test run time for most providers.  If needed, add an override method to the Swift provider.

lots of free time edition: we should not sleep at all.  Instead we should call `Utils.eventuallyTrue` which breaks as soon as the condition is met.  Most operations do not exhibit eventual consistency and I observed the most inconsistent provider exhibit it only 3% of the time:

https://github.com/andrewgaul/are-we-consistent-yet

I have seen eventual consistency take over 1,000 seconds to reconcile so bumping from 10 to 20 is not much of a guarantee.